controllers/release/release_suite_test.go

```diff
@@ -115,3 +118,26 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+// GetModuleCRDPath return the filesystem path after GOPATH/pkg/mod
+// for pseudo versioned modules.
+func GetModuleCRDPath(goModPath string, moduleName string) string {
+	var CRDPath string
+	readFile, err := os.Open(goModPath)
+	if err != nil {
+		panic(err)
+	}
+	scanner := bufio.NewScanner(readFile)
+	for scanner.Scan() {
+		re := regexp.MustCompile(".*" + moduleName + ".*")
+		match := re.Find([]byte(scanner.Text()))
+		if match != nil {
+			CRDPath = strings.Trim(string(match), "\t")
+			break
+		}
+	}
+	readFile.Close()
+	CRDPath = strings.ReplaceAll(CRDPath, " ", "@")
```

Comment:
Don't set the value just to return it afterwards. Also, this behavior should be documented. If not match is found or the scan doesn't work, you are just replacing text. Probably it's best to return `strings.ReplaceAll(strings.Trim(string(match), "\t")` directly inside the for loop. It would avoid you to having to use break and would move this line within the loop. In case of reaching this point, we would return `""` directly which is a healthier return IMHO. In addition, as we are moving to using defer, you don't need anything else after the loop.

Reply:
fixe.